package.json

```diff
@@ -119,6 +119,8 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@popperjs/core": "^2.11.6",
     "classnames": "^2.3.2",
+    "date-fns": "^2.11.0",
```

Comment:
Suggestion; can we avoid adding another date utility library given that the standard library is quite feature rich now, and we're already using moment elsewhere ?